src/views/layouts/includes/f-item.html

```diff
@@ -1,22 +1,29 @@
-<div class="f-item-group" id="{{@key}}">
+<div class="f-Item" id="{{@key}}">
 
-{{#if items}}
-	<div class="f-item-heading-group">
-		<h2 class="f-item-heading" data-f-toggle="labels">{{name}}</h2>
+	<div class="f-Item-header">
+		<h2 class="f-Item-heading" data-f-toggle="labels">{{name}}</h2>
```

Comment:
Note that, similar to https://github.com/cloudfour/fabricator/pull/17/files#diff-9d4bf388e581cd154c5ab375022023a9R17 , until `fabricator-assemble` integrates custom ordering of patterns, you'll ultimately want to use the `displayName` helper when rendering the `name` property to strip unwanted numeric prefixing from the `name`.